src/ui/components/CommandPalette/CommandPalette.tsx

```diff
@@ -47,7 +47,8 @@ export type CommandKey =
   | "jump_to_next_pause"
   | "jump_to_previous_pause"
   | "toggle_protocol_panel"
-  | "toggle_protocol_timeline";
+  | "toggle_protocol_timeline"
+  | "restart_session";
```

Comment:
Nit: I think we usually try to alpha sort lists like this (though the "jump_to" keys already didn't do this so 🤷🏼 )